src/main/java/EmptyLineException.java

```diff
@@ -0,0 +1,2 @@
+public class EmptyLineException extends Exception{
+}
```

Comment:
Formatting error for `Exception{`. Also, I'm not sure why your exception class is empty?